packages/jest-validate/src/utils.ts

```diff
@@ -22,6 +22,8 @@ export const format = (value: unknown): string =>
 export const formatPrettyObject = (value: unknown): string =>
   typeof value === 'function'
     ? value.toString()
+    : typeof value === 'undefined'
+    ? 'undefined'
```

Comment:
`JSON.stringify(undefined)` returns `undefined` (not the string - the value)

Reply:
@SimenB your build `29.3.1` is borked